core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -1286,7 +1286,7 @@ static function ($allowedTag)
 	private function parseUrlWithQueryString(string $url): array
 	{
 		// Restore [&] and &amp;
-		$url = str_replace(array('[&]', '&amp;'), '&', $url);
+		$url = str_replace(array('[&]', '&amp;', '&#61;'), array('&', '&', '='), $url);
```

Comment:
```suggestion
		$url = str_replace(array('&#61;', '[&]', '&amp;'), array('=', '&', '&'), $url);
```
As `&amp;#61;` should not get decoded to `=` I think?